### ISSUE TITLE
ribbonComboBox DropDownItemClicked and TextBoxTextChanged events not firing anymore

### ISSUE THREAD

**Issue by MaxFranz** [RibbonWinForms/RibbonWinForms#18]
hi, 
it seems that after some kind of Windows update in the last few weeks, DropDownItemClicked and TextBoxTextChanged  events of ribbonComboBox are not fired anymore (before they were working fine).
Unfortunately it doesn't happen on my develop machine but on few production machines (windows 7, 8 and 10)
Am I the only one with this issue?

Thanks

**Comment by kibitzerCZ** [comment]
I am experiencing a similar weird behavior on *some* Windows 10 computers. I have an OrbDropDown with a collection of menu items (RibbonButtons and RibbonSeparators). While on a majority of Win 10 computers everything works well, on a few computers with the *exact same* Windows 10 build *some* RibbonButtons from the menu bottom do not work - their Click event is not getting fired at all. I have added a checking code blocks to RibbonItem.OnClick method, but still no luck:

```csharp
/// <summary>
/// Raises the Click event
/// </summary>
/// <param name="e">Event data</param>
public virtual void OnClick(EventArgs e)
{
  if (!Enabled) 
  {
	//following testing messagebox is not displayed - this is expected as the item is enabled
	MessageBox.Show("item disabled");
	return;
  }

  if (ClosesDropDownAt(Canvas.PointToClient(Cursor.Position)))
  {
	DeactivateToolTip(_TT);
	RibbonPopupManager.Dismiss(RibbonPopupManager.DismissReason.ItemClicked);
  }

  SetSelected(false);

  if (Click != null)
  {
	//following messagebox displays list of two delegates on first computer, 
	//DOES NOT GET DISPLAYED ON THE FAILING COMPUTER
	MessageBox.Show( <delegate names>);	
    	Click(this, e);
  }
  else
	MessageBox.Show("no click delegates set");	//DOES NOT GET DISPLAYED ON THE FAILING COMPUTER NEITHER !?
}
```

**Comment by kibitzerCZ** [comment]
Well, it seems that my issue can be resolved by applying patch described in issue [#10](https://github.com/RibbonWinForms/RibbonWinForms/issues/10). Sorry for being so hasty and not going through all issues properly.
@MaxFranz did you give it a try?

**Comment by adriancs2** [comment]
This problem is fixed in latest version. You may try the nuget version.
Guide: How to add ribbon library via Nuget:
https://github.com/RibbonWinForms/RibbonWinForms/wiki
